Source/Scene/Cesium3DTile.js

```diff
@@ -285,34 +352,59 @@ define([
 
     /**
      * DOC_TBA
+     *
+     * @private
      */
     Cesium3DTile.prototype.isReady = function() {
         return this._content.state === Cesium3DTileContentState.READY;
     };
 
     /**
+     * When <code>true</code>, the tile can be refined to when doing replacement refinement.
+     * <p>
+     * A tile is refinable when all of its children's content is loaded.  If a child is does
+     * not have empty (such as for accelerating culling), its descendants with content must
+     * be loaded first.
+     * </p>
+     *
      * DOC_TBA
+     *
+     * @private
      */
-    Cesium3DTile.prototype.isRefinable = function() {
+    Cesium3DTile.prototype.isReplacementRefinable = function() {
         return this._numberOfUnrefinableChildren === 0;
     };
 
     /**
+     * When <true>true</code>, the tile's content has not be requested.
+     *
      * DOC_TBA
```

Comment:
It's not ready, see #3241

>  Replace Cesium3DTile#isReady function with a readonly Cesium3DTile.contentReady property. Add doc.
 Likewise, replace isReplacementRefinable and isContentUnloaded with readonly properties. Complete doc.